### PR TITLE
Remove repo check from example workflows

### PR DESCRIPTION
**Pull request by Crissman**
## Motivation

Enable checking of example github actions locally using `act`.

## Description of the changes

Remove the line that stops execution for example workflows if not in the `optuna/optuna-examples` repo.

### CODE REVIEW

**Comment by Crissman** [comment]
Addresses issue #4

**Comment by Crissman** [comment]
@nzw0301 Can you review this?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -47,6 +47,16 @@ Currently, you can run the following jobs:
 
 - `checks`
   - Checks the format
+- 'examples'
```
Comment (nzw0301) [inline CONTRIBUTING.md:50]:
```suggestion
- `examples`
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -47,6 +47,16 @@ Currently, you can run the following jobs:
...
+  - Run the examples
+
+To run a specific example job:
+
+```bash
+act -j examples -W path/to/example.yml/file
+```
+
+Usually, the example.yml file will be in the .github/workflows/ directory.
```
Comment (nzw0301) [inline CONTRIBUTING.md:59]: How about 

```suggestion
Usually, the example.yml file will be in the [`.github/workflows/`](.github/workflows/) directory.
```

or 

```suggestion
Usually, the example.yml file will be in the `.github/workflows/` directory.
```
?

**Review by nzw0301 — CHANGES_REQUESTED** [review]
Basically LGTM! I leave minor comments on the markdown file. I could not confirm whether or not the example command of `act` works because `act` does not work on my local machine, which is a laptop with Apple M1 Silicon.

**Comment by Crissman** [comment]
@nzw0301 Thanks. Suggestions reflected. PTAL

**Review by nzw0301 — APPROVED** [review]
LGTM. Thank you for your updates!

**Pull request merged**